cmd/version.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/combo/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
```

Comment:
Should we update adding this sub-command to the root chain at the rootCmd call site instead? That way the definition of the rootCmd and it's child commands are all present centrally.

Reply:
So basically moving all of the `rootCmd.AddCommand()` to the `Execute()` function?

Something like this is what I'm seeing in my mind's eye from what you're saying, correct me if I'm wrong:

```go
// ./cmd/root.go
// Execute executes the root command.
func Execute(log logr.Logger) error {
	if log != nil {
		rootLog = log
	}
	
	// NEW =======
	rootCmd.AddCommand(versionCmd)
	rootCmd.AddCommand(evalCmd)
	rootCmd.AddCommand(runCmd)
        // ===========
        
	rootCmd.CompletionOptions.DisableDefaultCmd = true

	return rootCmd.Execute()
}

```

Reply:
That sounds reasonable to me assuming that's where the rootCmd is being defined, although that might just be a global variable currently (vs. we're using a constructor function to build up a rootCmd, returning a cobra.Command explicitly). We can iron out the command codebase structure over time, so let's settle on the Execute function for now.

Reply:
Makes sense. Updated.